manifests/init.pp

```diff
@@ -27,6 +27,22 @@
 #   The ensure parameter to pass to the package resource
 # [*package_name*]
 #   The name of the package(s) to be installed
+# [*trusted_cert_dir*]
```

Comment:
Next it would be good to convert these to Puppet Strings format.

Reply:
converted